sdk/src/inputs.rs

```diff
@@ -0,0 +1,28 @@
+use std::io::{self, Cursor};
+
+use borsh::{BorshDeserialize, BorshSerialize};
+
+use crate::{account_info::LightAccountInfo, proof::ProofRpcResult};
+
+pub struct LightInputs {
```

Comment:
can we rename to `LightInstructionData`?

Reply:
Done